components/authz-service/server/v2/authz_test.go

```diff
@@ -105,6 +105,68 @@ func TestProjectsAuthorized(t *testing.T) {
 	})
 }
 
+func TestV2ProjectsAuthorized(t *testing.T) {
```

Comment:
Technically v2 is a beta, too, so instead of "TestV2ProjectsAuthorized" and "TestV2BetaProjectsAuthorized"
perhaps just "TestV2ProjectsAuthorized" and "TestV2_1ProjectsAuthorized" ?

Reply:
this will be a little inconsistent with our migration status naming (`Successful` for v2, `SuccessfulBeta1` for v2.1, but i think i'll start using `p` in place of the decimal in camelcase names like google does: https://cloud.google.com/apis/design/versioning